apps/demos/src/polyfills.ts

```diff
@@ -52,10 +52,12 @@
  *
  */
 
+import './zone-flags'; // Included before zone.js
```

Comment:
we need some way to load this only on environments, any clue how?

Reply:
Target configurations and multiple polyfill files. Or custom build scripts